appIndicator.js

```diff
@@ -326,6 +326,11 @@ class AppIndicators_IconActor extends St.Icon {
 
         Util.connectSmart(Gtk.IconTheme.get_default(), 'changed', this, '_invalidateIcon')
 
+        Util.connectSmart(this, 'enter-event', this, () => {
+            this.opacity = 255;
+        });
+        Util.connectSmart(this, 'leave-event', this, '_setOpacity');
```

Comment:
Use function without quotes

Reply:
Actually, there's no need to use `Util.connectSmart` here as you're connecting to the same object, so just use simple arrow-functions (there's no need to disconnect from self on destruction).